test/test_jit.py

```diff
@@ -0,0 +1,420 @@
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
+
+    @unittest.skipIf(not RUN_CUDA, "no CUDA")
+    def test_scriptmodule_MFCC(self):
+
+        class MyModule(torch.jit.ScriptModule):
+            def __init__(self):
+                super(MyModule, self).__init__()
+                self.module = transforms.MFCC()
+                self.module.eval()
+
+            @torch.jit.script_method
+            def forward(self, tensor):
+                return self.module(tensor)
+
+        tensor = torch.rand((1, 1000), device="cuda")
+
+        jit_out = MyModule().cuda()(tensor)
+        py_out = transforms.MFCC().cuda()(tensor)
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
+
+    @unittest.skipIf(not RUN_CUDA, "no CUDA")
+    def test_scriptmodule_MelSpectrogram(self):
+
+        class MyModule(torch.jit.ScriptModule):
+            def __init__(self):
+                super(MyModule, self).__init__()
+                self.module = transforms.MelSpectrogram()
+                self.module.eval()
+
+            @torch.jit.script_method
+            def forward(self, tensor):
+                return self.module(tensor)
+
+        tensor = torch.rand((1, 1000), device="cuda")
+
+        jit_out = MyModule().cuda()(tensor)
+        py_out = transforms.MelSpectrogram().cuda()(tensor)
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
+
+    def test_torchscript_BLC2CBL(self):
+        @torch.jit.script
+        def jit_method(tensor):
+            # type: (Tensor) -> Tensor
+            return F.BLC2CBL(tensor)
+
+        tensor = torch.rand((10, 1000, 1))
+
+        jit_out = jit_method(tensor)
+        py_out = F.BLC2CBL(tensor)
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
+
+    @unittest.skipIf(not RUN_CUDA, "no CUDA")
+    def test_scriptmodule_BLC2CBL(self):
+
+        class MyModule(torch.jit.ScriptModule):
+            def __init__(self):
+                super(MyModule, self).__init__()
+                self.module = transforms.BLC2CBL()
+                self.module.eval()
+
+            @torch.jit.script_method
+            def forward(self, tensor):
+                return self.module(tensor)
+
+        tensor = torch.rand((10, 1000, 1), device="cuda")
+
+        jit_out = MyModule().cuda()(tensor)
+        py_out = transforms.BLC2CBL()(tensor)
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
+
+    def test_torchscript_mu_law_encoding(self):
+        @torch.jit.script
+        def jit_method(tensor, qc):
+            # type: (Tensor, int) -> Tensor
+            return F.mu_law_encoding(tensor, qc)
+
+        tensor = torch.rand((10, 1))
+        qc = 256
+
+        jit_out = jit_method(tensor, qc)
+        py_out = F.mu_law_encoding(tensor, qc)
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
```

Comment:
Why are you changing the atol and rtol? I assume this is for floats. How did you choose these values? The default [is much stricter](https://pytorch.org/docs/stable/torch.html#torch.allclose).